nano/templates/account_management.tmpl

```diff
@@ -172,6 +172,7 @@
 	<br>
 	<hr>
 	{{if data.state == "payroll"}}
+		<span class='floatRight'>{{:helper.link('Set Passenger Rate', 'gear', {'action' : 'passenger_rate'}, data.auth != 1 ? 'disabled' : null)}}</span>
```

Comment:
This looks very odd here. Put this up after LN171 with an if gate for the "payroll" state (so it is before the line break and header